code/utils/calc_t.py

```diff
@@ -0,0 +1,23 @@
+
+import numpy.linalg as npl
+from scipy.stats import mstats
+from scipy.stats import t
+
+def significant(a,b,c):
+    df=len(b)-2
```

Comment:
can you comment on why this is len(b) -2 ? I know n-rank(x) is the degrees of freedom so why is x only rank 2? Maybe adding some comments can help as i am writing tests rn